tests/core/test_expectation_suite.py

```diff
@@ -185,6 +193,137 @@ def __dict__(self):
         )
 
 
+class TestCRUDMethods:
```

Comment:
I typically see our tests as just top-level functions; why introduce the class? I don't have a preference. just curious.

Reply:
i generally prefer top-level functions, but using the class is a good way to namespace. It lets the test names be shorter, which is useful, but in this case it helps make it clear that all of these tests are 1.0, and any other test in the file is legacy.